core/src/cubos/core/data/file.cpp

```diff
@@ -263,65 +263,83 @@ File::Handle File::find(std::string_view path)
 
 File::Handle File::create(std::string_view path, bool directory)
 {
-    // Remove trailing slashes.
-    while (path.ends_with('/'))
+    if (path.starts_with("./"))
     {
-        path.remove_suffix(1);
+        path.remove_prefix(2);
     }
```

Comment:
Should this be able to handle `././<rel path>`?

Reply:
I'll refactor this when I write tests for this code.